frontend/packages/console-shared/src/components/dashboard/utilization-card/top-consumer-popover.scss

```
@@ -0,0 +1,69 @@
+@import '../../../styles/skeleton-screen';
+
+.co-utilization-card-popover__dropdown, .co-utilization-card-popover__dropdown .pf-c-dropdown__toggle {
```

```suggestion
.co-utilization-card-popover__dropdown > div, .co-utilization-card-popover__dropdown .pf-c-dropdown__toggle {
```
if you add this I think you can remove line 7 right?

---

we want them both to be 100%.

---

this would make the div and the toggle 100%, does it need a width on `..co-utilization-card-popover__dropdown` too?